docs/memory_server_api.md

```
@@ -0,0 +1,641 @@
+### 3. 헬스 체크
+```bash
+# 서버 상태 확인
+curl http://memory-server/health
+
+# 데이터베이스 연결 확인
+curl http://memory-server/system/info
+```
+
+## 추가 리소스
+
+- **Swagger UI**: http://memory-server/docs
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)